geode-client-native/src/cppcache/impl/CppCacheLibrary.cpp

```diff
@@ -79,7 +84,6 @@ typedef ACE_Singleton<CppCacheLibrary,ACE_Recursive_Thread_Mutex> TheLibrary;
 // internals are prepared. fw_helper framework will handle this.
 CppCacheLibrary* CppCacheLibrary::initLib( void )
 {
-  ACE::init();
```

Comment:
Hi,

I'm not sure - which declaration are you referring to ?
and regarding the ACE bug - pls see my previous answer :)

Thanks,
Gal